src/components/MultiHopTrade/components/TradeInput/components/TradeQuotes/TradeQuotes.tsx

```diff
@@ -93,14 +156,71 @@ export const TradeQuotes: React.FC<TradeQuotesProps> = memo(({ isLoading, onBack
     isTradeQuoteRequestAborted,
     bestQuoteData?.quote,
     bestQuoteData?.inputOutputRatio,
-    tradeQuoteDisplayCache,
+    validTradeQuotesDisplayCache,
     activeQuoteMeta,
     isTradeQuoteApiQueryPending,
     isLoading,
     isSwapperQuoteAvailable,
     onBack,
   ])
 
+  const invalidQuotes = useMemo(() => {
+    if (isTradeQuoteRequestAborted) {
+      return []
+    }
+
+    const bestTotalReceiveAmountCryptoPrecision = bestQuoteData?.quote
+      ? getBuyAmountAfterFeesCryptoPrecision({
+          quote: bestQuoteData.quote,
+        })
+      : undefined
+
+    return invalidTradeQuotesDisplayCache.map((quoteData, i) => {
+      const { swapperName, id, errors, isStale } = quoteData
+
+      const isActive = activeQuoteMeta !== undefined && activeQuoteMeta.identifier === id
```

Comment:
question: is there ever a case we want an invalid quote to be active?

Reply:
Absolutely not, removed in [2c0e7de](https://github.com/shapeshift/web/pull/8087/commits/2c0e7ded59e68034b730c8481c951c1f9dbf0bfa)